Modules/SubModule_Import/xASL_imp_ReadSourceData.m

```diff
@@ -74,6 +74,9 @@ function xASL_imp_ReadSourceData_CheckFolderHierarchy(x)
     % Condition for file extension
     conditionFile = '.(dcm|ima|xml|par|rec)';
 
+    % Other extension
+    conditionExtension = '.';
```

Comment:
'.' is **any** character in regexpi notation. You want '\\.'

Reply:
c503604